File: python/ninety_nine_bottles.py
def ninety_nine_bottles():
    for i in range(99, 1, -1):
        first_line = "{} bottles of beer on the wall. ".format(str(i))
        print(first_line * 2, "\nTake one down, pass it around. {} {} of beer on the wall.".format(str(i - 1), "bottle" if (i - 1) == 1 else "bottles"))
        
ninety_nine_bottles()

File: java/ninetyNineBottles.java
class Main {
  public static void main(String[] args) {
    for (int beers = 99; beers > -1; beers--) {
      if (beers > 0) {
        System.out.println(String.format("%s %s of beer on the wall.", beers, (beers > 1 ? "bottles":"bottle")));
        System.out.println("Take one down, pass it around.");
        System.out.println(String.format("%s %s of beer on the wall\n.", beers - 1, (beers > 1 ? "bottles":"bottle")));
      } else { 
        System.out.println("Oh, no!  We're all out of beer!");
      }
    }
  }
}
